api/data_refinery_api/views.py

```diff
@@ -221,20 +221,23 @@ def get(self, request, format=None):
         filter_dict = request.query_params.dict()
         filter_dict.pop('limit', None)
         filter_dict.pop('offset', None)
+        order_by = filter_dict.pop('order_by', None)
         ids = filter_dict.pop('ids', None)
 
         if ids is not None:
             ids = [ int(x) for x in ids.split(',')]
             filter_dict['pk__in'] = ids
 
         samples = Sample.objects.filter(**filter_dict)
+        if order_by:
+            samples = samples.order_by(order_by)
 
         page = self.paginate_queryset(samples)
         if page is not None:
-            serializer = SampleSerializer(page, many=True)
+            serializer = DetailedSampleSerializer(page, many=True)
             return self.get_paginated_response(serializer.data)
         else:
-            serializer = SampleSerializer(samples, many=True)
+            serializer = DetailedSampleSerializer(samples, many=True)
```

Comment:
If I'm not mistaken, these are the only two places we used SampleSerializer. Is there any point to having it around anymore? Although if we go the route of nixing it, I would think we'd probably want to get rid of DetailedSampleSerializer and make SampleSerializer into what it was.

Reply:
I'm worried this is going to end up being very slow. They need it right now, okay, but let's hang on to the skinny one until we're sure we're not going to need it.